src/components/pie-chart/chart-tooltip-content.tsx

```diff
@@ -0,0 +1,407 @@
+//                 ref={ref}
+//                 className={cn(
+//                     'grid min-w-[8rem] items-start gap-1.5 rounded-lg border bg-tooltip-background-light px-3 py-2 text-xs shadow-xl',
+//                     className
+//                 )}
+//             >
+//                 {!isSinglePayload ? tooltipLabel() : null}
+//                 <div className="grid gap-1.5">
+//                     {payload.map((item, index) => {
+//                         const indicatorColor =
+//                             item.color || item.payload?.fill || color || '#000';
+
+//                         return (
+//                             <div
+//                                 key={item.dataKey || index}
+//                                 className={cn(
+//                                     'flex w-full items-stretch gap-2',
+//                                     indicator === 'dot' && 'items-center'
+//                                 )}
+//                             >
+//                                 {!hideIndicator && (
+//                                     <div
+//                                         className={cn({
+//                                             'h-2.5 w-2.5 ': indicator === 'dot',
+//                                             'w-1 h-full': indicator === 'line',
+//                                             'w-0 border-[0.5px] border-dashed':
+//                                                 indicator === 'dashed',
+//                                         })}
+//                                         style={{
+//                                             backgroundColor:
+//                                                 indicator === 'dot' || indicator === 'line'
+//                                                     ? indicatorColor as string
+//                                                     : '',
+//                                             borderColor:
+//                                                 indicator === 'dashed' ? indicatorColor as string : '',
+//                                         }}
+//                                     />
+//                                 )}
+//                                 <div className="flex-1 flex justify-between items-center">
+//                                     <span>{item[nameKey] || item.dataKey}</span>
+//                                     <span className="font-mono font-medium">
+//                                         {formatter ? formatter(item.value ?? '') : item.value ?? ''}
+//                                     </span>
+//                                 </div>
+//                             </div>
+//                         );
+//                     })}
+//                 </div>
+//             </div>
+//         );
+//     }
+// );
+
+// ChartTooltipContent.displayName = 'ChartTooltipContent';
+
+// export default ChartTooltipContent;
+
+import React, { ReactNode } from 'react';
```

Comment:
Don't forget to add `type` to the imported type.

```suggestion
import React, { type ReactNode } from 'react';
```